airbyte-integrations/connectors/source-netsuite/.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.11.9
```

Comment:
Remove this, please. Hypothetically no harm, but mostly we target Python 3.10+, and we don't want anything that won't run on 3.10 to make it into any given connector just yet.

Reply:
Thanks for catching this! I was trying to keep it out of my commits but must have done an add all.